packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -80,10 +126,10 @@ export const TokensList: React.FC<TokensListProps> = (props) => {
                 <TokenCard
                   key={`token${name}`}
                   category={tokenCategory}
-                  name={name}
+                  name={useJavascriptNames ? camelCase(name) : name}
                   value={value}
                   comment={comment}
-                  backgroundColor="rgb(255, 255, 255)"
+                  backgroundColor={selectedTheme === 'dark' ? 'rgba(15, 22, 33, 1)' : 'rgb(255, 255, 255)'}
```

Comment:
question: does `backgroundColor` take a token name as a value? Rather than passing hardcoded rgb colors should we be passing tokens?

Reply:
The `backgroundColor` needs to change based on the selected theme in the filter, rather than the current site-wide theme. If we give the `TokenCard` a token name as a value it'll display the color based on the site-wide theme.

I'll try to get the correct rgb value from the dist tokens file and pass it in that way (conditionally based on selected filter theme) so that it's more accurate if we ever change the token values.

Reply:
Ah-ha! Yeah that makes perfect sense and great suggestion to make this implementation better too